drivers/shared/executor/executor_universal_linux.go

```diff
@@ -173,19 +174,19 @@ func DestroyCgroup(groups *lconfigs.Cgroup, executorPid int) error {
 	return mErrs.ErrorOrNil()
 }
 
-func (e *UniversalExecutor) start(command *ExecCommand) error {
-	if command.NetworkIsolation != nil && command.NetworkIsolation.Path != "" {
+func wrapNetns(f func() error, spec *drivers.NetworkIsolationSpec) error {
```

Comment:
comment

Reply:
Coming with a functional background, I would have expected `wrapNetns` to return a function that wraps passed on.  Maybe `withinNetns` is more appropriate name?